uPortal-security/uPortal-security-authn/src/main/java/org/apereo/portal/security/provider/SimpleSecurityContext.java

```diff
@@ -53,6 +53,12 @@ public synchronized void authenticate() throws PortalSecurityException {
         this.isauth = false;
         if (this.myPrincipal.UID != null && this.myOpaqueCredentials.credentialstring != null) {
 
+            // Logs if an attempt is made to log into a local account
+            StringBuilder msg = new StringBuilder();
+            msg.append("An attempt to log into the local login has occurred. ");
+            msg.append("user=" + this.myPrincipal.UID);
+            if (log.isWarnEnabled()) log.warn(msg.toString()); 
+
```

Comment:
I would wrap the StringBuilder in the if statement.

Reply:
Thanks for the suggestion. I moved the message into the if statement and cleaned up the code. It now looks like the rest of the logging in this file. I originally built it so that if anyone wanted to add additional things to the log message (i.e. IP or something of that nature) they could.